internal/os/diskutil/types.go

```diff
@@ -0,0 +1,114 @@
+package diskutil
```

Comment:
This could be under https://github.com/kubernetes-csi/csi-proxy/blob/master/internal/os/disk/types.go